tasks/managed/close-advisory-issues/close-advisory-issues.yaml

```diff
@@ -0,0 +1,100 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: close-advisory-issues
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >
+    Tekton task to close all issues referenced in the releaseNotes. It is meant to run after
+    the advisory is published. A comment will be added to each closed issue with a link
+    to the advisory it was fixed in
+  params:
+    - name: dataPath
+      description: Path to the JSON string of the merged data to use in the data workspace
+      type: string
+    - name: advisoryUrl
+      description: The url of the advisory the issues were fixed in. This is added in a comment on the issue
+      type: string
+  workspaces:
+    - name: data
+      description: The workspace where the snapshot spec json file resides
+  steps:
+    - name: close-issues
+      image: quay.io/konflux-ci/release-service-utils:0f82be4be43294b6a96846d87ef7f7c0b9e34267
+      env:
+        - name: ACCESS_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: todo
```

Comment:
This should not be merged with the name and key as todo (e2e) won't pass anyways. It is also in the mocks. Waiting on Rogue to provide the key and releng to add it to update this

Reply:
Any update on this? Still waiting? Because Rogue is on PTO?

Reply:
Ryan started working on it Monday or so. Not blocked on Rogue, I just think Ryan is busy

Reply:
updated